osbs/api.py

```diff
@@ -521,14 +524,19 @@ def _check_labels(self, repo_info):
                       utils.Labels.LABEL_TYPE_VERSION]:
             try:
                 _, req_labels[label] = labels.get_name_and_value(label)
+
+                if label in strict_labels and not req_labels[label]:
```

Comment:
Could this block be in the try/except `else` clause?

Reply:
I'd rather leave it here as it is more visible than in else after except